### ISSUE TITLE
Pretty-print on `update` breaks files with `test>` lines

### ISSUE THREAD

**Issue by mambocab** [unisonweb/unison#4430]
To reproduce:

- In one terminal, start `ucm`.
- Add the following contents to `bug.u` in the directory watched by `ucm`:

```unision
add : Nat -> Nat -> Nat
add x y = x + 1

test> add.tests.e1 = check (add 1 2 == 3)
```

- in `ucm`, run `update`

Expected: `bug.u` still compiles and has the same meaning as before running `update`.

Actual: `load bug.u` fails with the following error:

```
quickstart/main> load bug.u

  offset=8:
  unexpected :
      2 | add.tests.e1 : [Result]
```

Because `bug.u` has been rewritten to the following:

```
add.tests.e1 : [Result]
add.tests.e1 : [Result]
add.tests.e1 =
  use Nat ==
  check (add 1 2 == 3)

add : Nat -> Nat -> Nat
add x y =
  use Nat +
  x + y

test> add.tests.e1 : [Result]
add.tests.e1 =
  use Nat ==
  check (add 1 2 == 3)

---

add : Nat -> Nat -> Nat
add x y = x + y

test> add.tests.e1 = check (add 1 2 == 3)
```

**Comment by aryairani** [comment]
Hi @mambocab, can you check if you still have this issue with the latest development build https://github.com/unisonweb/unison/releases/tag/latest

Otherwise, I believe it's fixed already in #4413

**Comment by mambocab** [comment]
Yep, seems to be fixed! Thanks for taking the time to confirm.
